### PR TITLE
Dilpreet |Stage | UI - Currect the GSTR 1 and GSTR 2 Design.

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
https://app.clickup.com/t/2nau7vb

### CODE REVIEW

**Comment by dilpreetsinghofficial** [comment]
Reviewed

**Comment by dilpreetsinghofficial** [comment]
Reviewed

**Review by the-prateek-mishra — APPROVED** [review]

File: apps/web-giddh/src/app/gst/modals/push-to-portal/push-to-portal.component.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { Component, EventEmitter, Input, Output, ChangeDetectionStrategy } from '@angular/core';
+@Component({
+    selector: 'push-to-portal',
+    templateUrl: './push-to-portal.component.html',
+    styleUrls: ['./push-to-portal.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PushtToPortalComponent {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/gst/modals/push-to-portal/push-to-portal.component.ts]:
```suggestion
export class PushToPortalComponent {
```

File: apps/web-giddh/src/app/gst/modals/push-to-portal/push-to-portal.component.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+     *
+     * @param {Event} event
+     * @memberof PushtToPortalComponent
+     */
+    public onDownloadExcel(event: Event): void {
+        this.downloadExcel.emit(event);
+    }
+
+    /**
+ * This will use for download success sheet
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/gst/modals/push-to-portal/push-to-portal.component.ts]: fix formatting

File: apps/web-giddh/src/assets/locale/gst/en.json [hunk 3]
```diff
@@ -108,7 +108,15 @@
         "transaction_date": "Transaction Date",
         "reconcilation_note": "All transactions in Giddh Books whose vendor GSTIN, invoice number, invoice date, invoice amount and tax amount are same as the transactions in GSTN are displayed in this section.",
         "cancel_title": "Are you sure you want to cancel it?",
-        "total_invoice_value": "Total Invoice Value"
+        "total_invoice_value": "Total Invoice Value",
+        "errors":"Errors",
+        "push_to_portal":"Push to Portal",
+        "gst_offline_utility":"GST Offline Utility",
+        "download_excel":"Download Excel",
+        "excel_file_support":"1. This Excel file support , GSTIN Portal Offline tool.",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/gst/en.json]:
```suggestion
        "excel_file_support":"1. This Excel file support GSTIN Portal Offline tool.",
```

File: apps/web-giddh/src/assets/locale/gst/en.json [hunk 4]
```diff
@@ -108,7 +108,15 @@
...
+        "organised_accountant":" 2. Its an organised way to give to any accountant/CA.",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/gst/en.json]:
```suggestion
        "organised_accountant":"2. It's an organised way to give to any accountant/CA.",
```

File: apps/web-giddh/src/assets/locale/gst/en.json [hunk 5]
```diff
@@ -108,7 +108,15 @@
...
+        "json_support":"JSON file support by GST Portal",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/gst/en.json]:
```suggestion
        "json_support":"JSON file supported by GST Portal",
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Comment by dilpreetsinghofficial** [comment]
Reviewed

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**